bootstrap.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+sudo yum -y groupinstall 'Development Tools'
+sudo yum -y install tcl 
```

Comment:
@olivermel Can you add jemalloc and jemalloc-devel to this list of packages to install?  It seems those are not installed on our servers, but are required to build.